jquery-toggle.js

```diff
@@ -53,6 +53,17 @@
 		button.css('min-width', max_width + 'px');
 	}
 
+	/*
+	 * Helper function to dispatch change event.
+	 */
+	function simulateChange(elem) {
+		const event = new Event('change', {
```

Comment:
Argh, I think this would benefit from an IE 11 support, as we should assume that this file is not build using babel for now.

Relatively simple way to support older browsers https://stackoverflow.com/a/42089476, where the init should probably disable cancelling ` event.initEvent('change', true, false);`